ast/annotations_test.go

```diff
@@ -250,8 +250,8 @@ p = 1`,
 			note: "package inherits super-package",
 			modules: []string{
 				`# METADATA
-# title: root
 # scope: subpackages
+# title: root
```

Comment:
I might be missing something -- Is this a requirement that needs documentation? i.e. "scope comes first"?

Reply:
No, the ordering among annotations isn't relevant. I just put `scope` on top as that is more relevant to what is being tested; easier to spot.